src/OperationalInsights/OperationalInsights/OperationalInsights.format.ps1xml

```diff
@@ -333,7 +333,7 @@
                 <PropertyName>Statistics</PropertyName>
               </ListItem>
               <ListItem>
-                <ScriptBlock>ConvertTo-Json $_.Error -Depth 10</ScriptBlock>
+                <ScriptBlock>$_.Error ? (ConvertTo-Json $_.Error -Depth 10) : $_.Error</ScriptBlock>
```

Comment:
When Error is null, the behavior of Error and Statistics properties will be maintained.

Reply:
Please use if-else because `?:` is only supported in PowerShell 7. Thanks

Reply:
Done